client/src/main/java/io/pravega/client/stream/impl/PravegaCredsWrapper.java

```diff
@@ -32,12 +32,13 @@ public String getAuthenticationType() {
     public Map<String, List<String>> getRequestMetadata(URI uri) throws IOException {
         Map<String, String> metadata = creds.getAuthParameters();
 
-        Map<String, List<String>> retVal = metadata.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey,
-                e -> {
-                    List<String> list = Collections.singletonList(e.getValue());
-                    return list;
-                }));
-        retVal.put("method", Collections.singletonList(creds.getAuthenticationType()));
+        Map<String, List<String>> retVal = metadata.entrySet().stream().collect(
+                Collectors.toMap(entry -> Credentials.AUTH_HANDLER_PREFIX + entry.getKey(),
+                        e -> {
```

Comment:
These 3 lines can probably be replaced with `Collections::singletonList`

Reply:
Fixed.